### ISSUE TITLE
[Bug] Hight accuracy, low loss but mAP very very low (~0)

### ISSUE THREAD

**Issue by vietthang2502** [open-mmlab/mmdetection#9787]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmdetection/issues) and [Discussions](https://github.com/open-mmlab/mmdetection/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmdetection.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (master)](https://github.com/open-mmlab/mmdetection) or [latest version (3.x)](https://github.com/open-mmlab/mmdetection/tree/dev-3.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

master branch https://github.com/open-mmlab/mmdetection

### Environment

.

### Reproduces the problem - code sample

.

### Reproduces the problem - command or script

![Screenshot from 2023-02-16 16-48-10](https://user-images.githubusercontent.com/77373839/219329869-37fe180d-34d7-4d07-9bae-0684131fbcdd.png)
![Screenshot from 2023-02-16 16-48-24](https://user-images.githubusercontent.com/77373839/219329928-d7a99428-bbb4-4859-aa45-f91099def220.png)

### Reproduces the problem - error message

.

**Comment by RangiLyu** [comment]
Please provide more details.
One possible reason is that your model is overfitting your training dataset.

**Comment by vietthang2502** [comment]
@RangiLyu 
i train on faster_rcnn_r50_fpn_1x_coco
i used LISA trafic light dataset and convert to COCO format
i try to train on very model in your opensource but the result always very bad like this. 
I reverse the problem, go from hyperparam to the model and then think about the problem in the dataset.
But i cant solve. Help me to solve it.
i was a newbie so practical exp is still immature :((((

**Comment by BIGWangYuDong** [comment]
have a look at your annotation file, and make sure your categories sequence is same

for example, in Objects365v1, it may get the same question if do not sort `categories`
https://github.com/open-mmlab/mmdetection/blob/c14dd6c42efb63f662a63fe403198bac82f47aa6/mmdet/datasets/objects365.py#L99-L100

**Comment by BIGWangYuDong** [comment]
one more thing is to validate your training image, and show them to check whether is normal

**Comment by vietthang2502** [comment]
i think that my problem
my annotation json file have bbox
![Screenshot from 2023-02-20 17-48-16](https://user-images.githubusercontent.com/77373839/220084920-fcf84b2c-011f-4c35-9399-1e8d381daae3.png)

but when i show the dataloaders,
bbox is none (ex: in the picture num_bbox !=0)
how to solve it
![Screenshot from 2023-02-20 17-49-41](https://user-images.githubusercontent.com/77373839/220084936-3559fe18-87f7-47c0-ba21-732f00a218f3.png)

**Comment by hhaAndroid** [comment]
@vietthang2502 please refer to https://mmyolo.readthedocs.io/zh_CN/dev/get_started/15_minutes_object_detection.html
